posthog/models/event/sql.py

```diff
@@ -377,7 +377,15 @@
 SELECT DISTINCT event FROM events where team_id = %(team_id)s AND event NOT IN ['$autocapture', '$pageview', '$identify', '$pageleave', '$screen']
 """
 
-GET_EVENTS_VOLUME = "SELECT event, count() AS count, max(timestamp) AS last_seen_at FROM events WHERE team_id = %(team_id)s AND timestamp > %(timestamp)s GROUP BY event ORDER BY count DESC"
+GET_EVENTS_VOLUME = """
```

Comment:
Would be interesting if the aggregating mergetree could remove the need for updating these in the background task